app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,55 @@
+class InvitationsController < ApplicationController
+  before_action :get_resources, except: [:confirm]
+  skip_before_action :authenticate_user, only: [:confirm]
+
+
+  def new
+    authorize Invitation
+    @invitation = @todo_list.invitations.new
```

Comment:
Jeśli już tworzysz nowy obiekt to warto go od razu autoryzować (tzn. zamienić tutaj linijkę 7 i 8, po czym przekazać obiekt). Teraz tego nie używasz, ale w przyszłości możesz chcieć :)